playbooks/roles/consul/defaults/main.yml

```diff
@@ -42,3 +42,6 @@ consul_performance_raft_multiplier: 5
 consul_service_config: ""
 
 consul_url: "https://releases.hashicorp.com/consul/{{ consul_version }}/consul_{{ consul_version }}_linux_amd64.zip"
+
+consul_http_public_port: null
```

Comment:
Should we let the public port default to 18500 and then check for the password to decide to run the nginx proxy? We could then remove occurrences of `consul_http_public_port` in the host variables, and generally have a standard public port that can be easily changed in one place.

Reply:
Sounds good as well.  My original reasoning was that someone might want to expose the API without setting a password, but we certainly don't, and it's probably not a good idea to even support it, since it's rather dangerous, so I made the change you proposed.